conf/app.transport.equipment.routes

```diff
@@ -1,6 +1,11 @@
 ->         /                                                              app.transport.equipment.index.Routes
 
-GET        /transport-equipment/add-transport-equipment/:departureId      controllers.transport.equipment.AddTransportEquipmentYesNoController.onPageLoad(departureId: String, mode: Mode = NormalMode)
-POST       /transport-equipment/add-transport-equipment/:departureId      controllers.transport.equipment.AddTransportEquipmentYesNoController.onSubmit(departureId: String, mode: Mode = NormalMode)
-GET        /transport-equipment/add-transport-equipment/:departureId      controllers.transport.equipment.AddTransportEquipmentYesNoController.onPageLoad(departureId: String, mode: Mode = CheckMode)
-POST       /transport-equipment/add-transport-equipment/:departureId      controllers.transport.equipment.AddTransportEquipmentYesNoController.onSubmit(departureId: String, mode: Mode = CheckMode)
+GET        /transport-equipment/add-transport-equipment/:departureId                                                                controllers.transport.equipment.AddTransportEquipmentYesNoController.onPageLoad(departureId: String, mode: Mode = NormalMode)
+POST       /transport-equipment/add-transport-equipment/:departureId                                                                controllers.transport.equipment.AddTransportEquipmentYesNoController.onSubmit(departureId: String, mode: Mode = NormalMode)
+GET        /transport-equipment/add-transport-equipment/:departureId                                                                controllers.transport.equipment.AddTransportEquipmentYesNoController.onPageLoad(departureId: String, mode: Mode = CheckMode)
+POST       /transport-equipment/add-transport-equipment/:departureId                                                                controllers.transport.equipment.AddTransportEquipmentYesNoController.onSubmit(departureId: String, mode: Mode = CheckMode)
```

Comment:
Don't think this is an issue with your PR (might be worth fixing in this one), but I think this should be:

`GET /transport-equipment/change-add-transport-equipment/:departureId controllers.transport.equipment.AddTransportEquipmentYesNoController.onPageLoad(departureId: String, mode: Mode = CheckMode)`
`POST /transport-equipment/change-add-transport-equipment/:departureId controllers.transport.equipment.AddTransportEquipmentYesNoController.onSubmit(departureId: String, mode: Mode = NormalMode)`